mantidproject/autoreduce
# Added new WISH reduce script

Fixes #234 

To Test:

* Send a job on WISH and confirm that it completes and that data can be found in the output directory
* Download the script and run it on it's own, confirm that it runs and that data is in the specified directory
